app/src/app/api/v0/inventory/[inventory]/results/route.ts

```diff
@@ -1,37 +1,39 @@
 import UserService from "@/backend/UserService";
-import { db } from "@/models";
-import { apiHandler } from "@/util/api";
-import { NextResponse } from "next/server";
-import { getEmissionResults } from "@/backend/ResultsService";
+import {db} from "@/models";
```

Comment:
Run `prettier` on this file please and configure it to format on save if you can.

Reply:
I think this file just has formatting/ whitespace changes, so maybe just remove it from the PR if there is nothing else...